src/modules/balance/balance_ckb_model.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiModel, ApiModelProperty } from "swagger-express-ts";
+
+@ApiModel({
+  description: "Ckb balance",
+  name: "BalanceCkbModel",
+})
+export default class BalanceCkbModel {

```

Comment:
Extracting an interface would be helpful for front-end

```ts
// replace template string when TypeScript 4.1 released
type NumericHex = string;

interface IBalanceCkbModel {
  free: NumericHex ;
  occupied: NumericHex ;
  locked_order: NumericHex;
}



class BalanceCkbModel implements IBalanceCkbModel  {
 ...
}
```

Reply:
Okay, I'll change it to implements.